docs/js/contribution.md

```diff
@@ -0,0 +1,173 @@
+  - Testing framework
+  - NPM audit (vulnerability checks)
+  - License check tool
+
+
+### What's on them?
+
+#### Delivery plan
+  - What features
+  - Oder of feature delivery (so that we can anticipate PR sizes... else?)
+  - Basic architecture - some diagram and involved services
+  - Modules structure (CC, re-use module artifacts)
+
+
+- Setting up OS process?
+  - Coding and testing the feature?
+  - Aligning on release strategy?
+
+## Coding convention
+
+### Contribution programming language
+
+Contribution should happen in Typescript and endorse type-safety whenever possible.
+
+### API alignment
+
+To maintain a consistent API strategy, contribution and SDK teams align on the API. Final approval is on the SDK team.
+
+### Repository
+
+    - Repository is created on Github under SAP Organization by contribution team
+    - SDK and contribution team co-own the repository and have full access rights to it
+    - SDK team provides repository structure if needed. It's preferable in most cased.
+    - SDK team can audit repository settings and require reasonable updates to configuration
+
+### Introducing dependencies
+
+    - Dependency audit should happen as a part of PR review
+    - SDK provides a guide on dependency choices to the contributing team
+
+### Coding convention
+
+    - Contribution team should use coding convention of the SDK unless it's explicitly approved to be otherwise
+    - Most of coding convention rules should be enforced by automated code checkers and linters like ESLint and Prettier
+    - Enforcement happens via continuous integration pipeline
+    - SDK team provides guidelines on coding conventions that can't be automatically enforced
+    - SDK team shares documentation on standard patterns chosen by SDK for ... (what patterns do we use?)
+
+### Pull requests
+
+    - Pull request should contain some logical and complet pile of work
+    - Pull request should be **small**  for easy review
+    - Preferably Pull request should contain tangible incremental value
+    - Pull request are mandatorily reviewed by at least one of SDK team members
+
+### Branching strategy
+
+  - `Main` branch has to be always **release ready**. In other words: up to date, tested, and documented
+
+## Documentation
+
+SAP Cloud SDK for JavaScript has two types of documentation:
+
+    - Documentation portal
+    - Generated API documentation
+
+### Documentation portal
+
+    - After reaching `Beta`, feature or API library has to be documented by contribution team on SDK's documentation portal
+    - Documentation has to be kept up to date by contribution team
+
+### Generated API documentation
+
+    - API documentation has to be generated
+    - SDK team provides means to generate API documentation on the available code base
+
+### README
+
+    - Contribution team maintains `README` on the contribution repository according to development needs
+
+## Releases
+
+### Release process
+  - How versioning happens with SDK (semver)
```

Comment:
This section seems to be a bit unfinshed.

Reply:
I pushed updates with my latest edits.